### PR TITLE
chore: enable back cypress

### PR DESCRIPTION
**Pull request by keellyp**
## Context

This PR should enable back E2E tests with Cypress latest version.

## Description

- [x] Valid tests
- [x] Enable CI 
- [x] Enable screenshots and videos uploads for failing tests

Fixes [ISSUE-397](https://linear.app/getlago/issue/ISSUE-397/[project]-enable-back-e2e-tests)

### CODE REVIEW

File: cypress/e2e/00-auth/t0-signup.cy.ts [hunk 1]
```diff
@@ -5,21 +5,18 @@ describe('Sign up page test', () => {
...
-  )
+  it('should create a new user and redirect to the home after signup', () => {
+    cy.visit('sign-up')
+
+    cy.get('input[name="organizationName"]').type('Company name')
+    cy.get('input[name="email"]').type(userEmail)
+    cy.get('input[name="password"]').type(userPassword)
+    cy.get('[data-test="submit-button"]').click()
+
+    // TODO: Why is it redirecting to customer page and why is it forbidden?
```
Comment (ansmonjol) [inline cypress/e2e/00-auth/t0-signup.cy.ts:16]: Should this be removed? I guess what you describe here is related to permissions, but it should not happen with a newly created account 🤔

File: cypress/support/reusableConstants.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 export const planWithChargesName = `a plan with charges`
+export const planWithChargeCode = `a_plan_with_chargesnew_code_plan_with_charge`
```
Comment (ansmonjol) [inline cypress/support/reusableConstants.ts:2]: Any chance it's a wrong paste in the string and the value should be shorter?

Seems we pasted next to an existing value trying to replace it, imo 😬

**Review by ansmonjol — APPROVED** [review]
Few minor comments, approved in the meantime so I don't block you!

File: cypress/support/reusableConstants.ts (hunk 2 above)
Comment (keellyp) [reply cypress/support/reusableConstants.ts:2]: nice catch

**Pull request merged**